WordPress/Classes/Extensions/AbstractPost+Media.swift

```diff
@@ -0,0 +1,32 @@
+
+extension AbstractPost {
+
+    func fetchRemoteVideoURL(for media: Media, completion: @escaping ( Result<(videoURL: URL, posterURL: URL?)> ) -> Void) {
```

Comment:
I don't think this is the best place for this method because of:

- The method isn't related to the Post object in anyway, if we are doing an extension maybe it should be to the Media object.
- Even as an extension to the media object it will be a bit out of place because it's creating a MediaService inside and the entity objects should avoid creating service, it should be the other way around.

What about creating a helper class? What was the initial reason to move it here?

Reply:
Actually I don't have a strong reason, I was hesitant about putting it here too. Yes I can put that into a helper